### PR TITLE
Gh-1462 GetSchema

### PR DESCRIPTION
**Pull request by m607123**

### CODE REVIEW

**Comment by m607123** [comment]
Closing while tests are altered

File: core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/GetSchemaHandler.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * A {@code GetSchemaHandler} handles the {@link uk.gov.gchq.gaffer.store.operation.GetSchema} operation.
+ */
+public class GetSchemaHandler implements OutputOperationHandler<GetSchema, Schema> {
+    @Override
+    public Schema doOperation(final GetSchema operation, final Context context, final Store store) throws OperationException {
+        final Schema schema;
+        if (null == operation) {
+            return new Schema();
```
Comment (t616178) [inline core/store/src/main/java/uk/gov/gchq/gaffer/store/operation/handler/GetSchemaHandler.java]: Does it make sense to return a brand new Schema object if the input operation was null?

**Comment by m607123** [comment]
Closing while test fixed

File: core/store/src/test/java/uk/gov/gchq/gaffer/store/operation/handler/GetSchemaHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+    @Before
+    public void setup() {
+        handler = new GetSchemaHandler();
+        store = mock(Store.class);
+        context = mock(Context.class);
+        user = mock(User.class);
+        properties = new StoreProperties();
+        schema = new Schema.Builder()
+                .edge(TestGroups.EDGE, new SchemaEdgeDefinition.Builder()
```
Comment (p013570) [inline core/store/src/test/java/uk/gov/gchq/gaffer/store/operation/handler/GetSchemaHandlerTest.java]: The compact schema is very similar to the normal schema except for a few fields have been removed, such as the description field. Please could you add a description field to this schema. And add an assertNotEquals to check the compactSchema is different to the schema.

**Comment by p013570** [comment]
Closing while changes are made.

**Pull request merged**